tests/EventLogging.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+# 
+# LSST Data Management System
+#
+# Copyright 2008-2014  AURA/LSST.
+# 
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# 
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the LSST License Statement and 
+# the GNU General Public License along with this program.  If not, 
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+#
+# This tests the EventLogging mechanism; this is very similar to the testLog.py
+# file in pex_logging, but uses EventLog instead of ScreenLog.  
+# The events that are  sent are retrieved.
+#
+
+import unittest
+
+import os
+import lsst.pex.logging as log
```

Comment:
This was purely temporary, and has already been replaced in a subsequent ticket where pex_logging was completely removed.